apps/communications/contacts/js/activities.js

```diff
@@ -233,6 +240,10 @@ var ActivityHandler = {
     }
 
     if (this.activityDataType.indexOf('text/vcard') !== -1) {
+      if (fb.isFbContact(theContact)) {
+        Contacts.confirmDialog(null, 'facebook-export-forbidden', dismiss);
```

Comment:
closing this PR as we are going to implement it as suggested by Francisco. ongoing work is being done at PR #27419